app/helpers/bootstrap_form_builder.rb

```diff
@@ -0,0 +1,21 @@
+class BootstrapFormBuilder < ActionView::Helpers::FormBuilder
+  %w[text_field text_area file_field].each do |method_name|
+    define_method(method_name) do |attribute, options = {}|
+      feedback_name = @object_name + '-' + attribute.to_s + '-feedback'
+      options["aria-describedby"] = feedback_name
+      options[:class] = options[:class].present? ? "#{options[:class]} form-control" : 'form-control'
+      unless @object[attribute].nil?
+        options[:class] += (@object.errors[attribute].any? ? ' is-invalid' : ' is-valid')
+      end
+
+      @template.content_tag :div, class: 'form-group' do
+        label(attribute, class: 'form-label') + super(attribute, options) +
+          @template.content_tag(:div, class: 'invalid-feedback', id: feedback_name) do
+            @template.content_tag(:ul) do
+              @template.content_tag(:li, @object.errors[attribute].join(' and '))
```

Comment:
Šeit ari baigi nepatikasm join(' and '). Piemēram, jā kļudas ir divas, vai viena, tad viss ok. Bet jā kļudu ir desmit?
Ir tads helperis lai skaisti tekstu savienot:
https://api.rubyonrails.org/classes/ActionView/Helpers/OutputSafetyHelper.html#method-i-to_sentence

Bet tev tomer naksies pastradat un renderet katru erroru atseviškā <li> tagā. Un gadijumā jā tev kaut kur vajadzes errorus vienā rindā - tur tu varesi nostilet ar css.
 ```ruby
 @template.tag.ul do
        @object.errors[attribute].map do |error_message|
          @template.tag.li error_message
        end.join('').html_safe
      end
```